API/Services/MetadataService.cs

```diff
@@ -27,6 +29,14 @@ public async Task<StoredMetadata> GetMetadata(string name)
         {
             return await Task.FromResult(_storedMetadata.Find(data => data.Name == name).FirstOrDefault());
         }
+
+        public async Task<Collection<StoredMetadata>> GetAllMetadata()
+        {
+            var result = await Task.FromResult(_storedMetadata.Find(metadata => true).ToList()).ConfigureAwait(false);
```

Comment:
used to avoid forcing the callback to be invoked on the original context or scheduler, increases performance.

https://devblogs.microsoft.com/dotnet/configureawait-faq/ 
https://docs.microsoft.com/nb-no/dotnet/fundamentals/code-analysis/quality-rules/ca2007

Reply:
Sounds smart!